packages/yoroi-extension/app/ergo-connector/components/signin/CardanoSignTxPage.js

```diff
@@ -206,18 +197,17 @@ class SignTxPage extends Component<Props, State> {
     }
 
     const numberOfDecimals = tokenInfo ? tokenInfo.Metadata.numberOfDecimals : 0;
-    const shiftedAmount = request.entry.amount.shiftedBy(- numberOfDecimals);
-    const ticker = tokenInfo ? this.getTicker(tokenInfo)
+    const shiftedAmount = request.entry.amount.shiftedBy(-numberOfDecimals);
+    const ticker = tokenInfo
+      ? this.getTicker(tokenInfo)
       : assetNameFromIdentifier(request.entry.identifier);
 
     let fiatAmountDisplay = null;
 
+    // this is a feature flag
```

Comment:
When I enable it looks like this!
<img width="656" alt="Screen Shot 2022-10-06 at 1 32 02 PM" src="https://user-images.githubusercontent.com/72753578/194302061-545bc3e5-6412-4784-961a-fe0bc5fb2320.png">

Reply:
No on the scope of this task, just old code that is there (you can see the forced `false`). This file was formatted, linted and I removed the old code that was used to show the collateral message, not done anything else :smile:

Reply:
Got it! I though you should be updating this as well

Reply:
<details>
<summary> As I might forget to fix it later. I fixed here 😅 </summary>
<img width="485" alt="Screen Shot 2022-10-06 at 1 58 42 PM" src="https://user-images.githubusercontent.com/72753578/194307356-17f62d29-38e4-40f0-8de4-f978310b978a.png">

</details>